app/controllers/pxe_controller/pxe_image_types.rb

```diff
@@ -85,37 +85,33 @@ def pxe_image_type_form_field_changed
   # Common VM button handler routines
   def pxe_image_type_button_operation(method, display_name)
     pxes = []
```

Comment:
I'd love to use here something like:

```ruby
107     # Tries to load a single checked item on from params.                                                                         
108     # If there's none, takes the id sent in params[:id].                                                                          
109     #                                                                                                                             
110     # Returns:                                                                                                                    
111     #   id of the item as a Fixnum                                                                                                
112     #                                                                                                                             
113     def checked_or_params_id                                                                                                      
114       objs = find_checked_items                                                                                                   
115       obj = objs.blank? && params[:id].present? ? params[:id] : objs[0]                                                           
116       obj = from_cid(obj) if obj.present?                                                                                         
117       obj                                                                                                                         
118     end  
```

but in this case, multiple records can be checked. Would have to write a new method for this.

@martinpovolny are you OK with these changes?